sanity-io/structured-content-2022
# Session details: one/two speakers link to respective speaker pages

# Session details: one/two speakers link to respective speaker pages

## Intent

![image](https://user-images.githubusercontent.com/6001131/163990454-d24085c4-7679-4cb2-8171-855fb38b30b0.png)
When rendering a Session, makes the speaker images & text in one/two speaker(s) view link to respective speaker's page.

Solves [this](https://app.shortcut.com/sanity-io/story/18003/session-details-make-speakers-clickable) Shortcut story.

## Description

Also refactored a bit to use existing queries and `getEntityPath` instead of hard-coding `href`s.

## Testing this PR

1. Go to [/program/cross-functional-collaboration-for-structured-content](https://structured-content-2022-web-git-session-details-clickabl-51527e.sanity.build/program/cross-functional-collaboration-for-structured-content)
2. Verify that the images of Simen and Carrie link to their respective Speaker pages
3. Fire up Studio and open http://localhost:3333/desk/session;26a01e49-20ef-4514-983f-bff19428a13e
4. Add a third Speaker to the Session's list of speakers, and verify that the Speaker list renders each element with the correct links to the respective Speaker's pages
